VULNERABILITY_RESPONSE_PROCESS.md

```diff
@@ -0,0 +1,143 @@
+      - ii. Include systems/apps effected
+      - iii. Include solutions (if any) if patch cannot be applied
+    - e. Release date is discussed
+
+9. At release date, Response Team coordinates with developers to finalize update:
+    - a. Response Manager propagates the "hotfix branch" to trunk
+    - b. Response Manager includes vulnerability announcement draft in release notes
+    - c. Proceed with the Point or Regular Release
+
+## IV. Post-release Disclosure Process
+
+1. Response Team has 90 days to fulfill all points within section III
+
+2. If the Incident Response process in section III is successfully completed:
+    - a. Response Manager contacts researcher and asks if researcher wishes for credit
+    - b. Finalize vulnerability announcement draft and include the following:
+      - i. Project name and URL
+      - ii. Versions known to be affected
+      - iii. Versions known to be not affected (for example, the vulnerable code was introduced in a recent version, and older versions are therefore unaffected)
+      - iv. Versions not checked
+      - v. Type of vulnerability and its impact
+      - vi. If already obtained or applicable, a CVE-ID
+      - vii. The planned, coordinated release date
+      - viii. Mitigating factors (for example, the vulnerability is only exposed in uncommon, non-default configurations)
+      - ix. Workarounds (configuration changes users can make to reduce their exposure to the vulnerability)
+      - x. If applicable, credits to the original reporter
+    - c. Release finalized vulnerability announcement on website and reddit /r/Monero
+    - d. For HIGH severities, release finalized vulnerability announcement on well-known mailing lists:
+      - i. oss-security@lists.openwall.com
+      - ii. bugtraq@securityfocus.com
+    - e. If applicable, developers request a CVE-ID
+      - i. The commit that applied the fix is made reference too in a future commit and includes a CVE-ID
+
+3. If the Incident Response process in section III is *not* successfully completed:
+    - a. Response Team and developers organize an IRC meeting to discuss why/what points in section III were not resolved and how the team can resolve them in the future
+    - b. Any developer meetings immediately following the incident should include points made in section V
+    - c. If disputes arise about whether or when to disclose information about a vulnerability, the Response Team will publicly discuss the issue via IRC and attempt to reach consensus
+    - d. If consensus on a timely disclosure is not met (no later than 90 days), the researcher (after 90 days) has every right to expose the vulnerability to the public
+
+## V. Incident Analysis
+
+1. Isolate codebase
+    - a. Response Team and developers should coordinate to work on the following:
+      - i. Problematic implementation of classes/libraries/functions, etc.
+      - ii. Focus on apps/distro packaging, etc.
+      - iii. Operator/config error, etc.
```

Comment:
>ii. Focus on apps/distro packaging, etc.

Well, since we distribute Docker and snap, and provide systemd + conf files, I don't think it's entirely out of scope.